src/Components/WebAssembly/testassets/HostedInAspNet.Client/HostedInAspNet.Client.csproj

```diff
@@ -1,12 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <OutputType>Exe</OutputType>
-    <ReferenceBlazorBuildLocally>true</ReferenceBlazorBuildLocally>
-    <RazorLangVersion>3.0</RazorLangVersion>
-    <!-- Disable compression in this project so that we can validate that it can be disabled -->
-    <BlazorEnableCompression>false</BlazorEnableCompression>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
```

Comment:
Did we get around the publish hosted bit by forcing the RID?

Reply:
I thought I'd tried this, but there's an issue where `dotnet publish -r [rid]` doesn't work. The restore that runs as part of publish messes things up. I'll talk to SDK folks about this and see what they recommend doing here.